src/main/java/picard/sam/markduplicates/MarkDuplicatesForFlowHelper.java

```diff
@@ -177,6 +177,7 @@ public ReadEndsForMarkDuplicates buildReadEnds(final SAMFileHeader header, final
         ends.read1Coordinate = getReadEndCoordinate(rec, !rec.getReadNegativeStrandFlag(), true, md.flowBasedArguments);
         if (md.flowBasedArguments.FLOW_USE_END_IN_UNPAIRED_READS) {
             ends.read2Coordinate = getReadEndCoordinate(rec, rec.getReadNegativeStrandFlag(), false, md.flowBasedArguments);
+            ends.read2CoordinateRequiresSerialize = true;
```

Comment:
Can you add a comment explaining why this is necessary here for future reference?

Reply:
done